bool turn_off
---
bool result